assets/src/components/pre-publish-panel.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * WordPress dependencies
+ */
+import { Notice } from '@wordpress/components';
+import { PluginPrePublishPanel } from '@wordpress/edit-post';
+import { Fragment } from '@wordpress/element';
+import { withSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Conditionally adds a notice to the pre-publish panel for the featured image.
+ *
+ * @return {Function} Either a plain pre-publish panel, or the panel with a featured image notice.
+ */
+const PrePublishPanel = ( { featuredMedia, validationCallback, missingMediaMessage, invalidMediaMessage, status } ) => {
+	if ( featuredMedia && validationCallback( featuredMedia ) ) {
```

Comment:
@swissspidy The `featuredMedia` here is the attachment ID, but it is getting passed into `hasMinimumFeaturedImageWidth` which expects a media object.

Reply:
Thanks! Somehow I missed this.